src/win/pty.cc

```diff
@@ -288,13 +285,13 @@ static NAN_METHOD(PtyStartProcess) {
    int result = winpty_start_process(pc, shellpath.c_str(), cmdline, cwd, env);
    if(result != 0) {
       why << "Unable to start terminal process. Win32 error code: " << result;
-      exception = NanThrowError(why.str().c_str());
```

Comment:
`NanThrowError` doesn't return anything in [1.7](https://github.com/rvagg/nan#nanthrowerrormessage-nanthrowtypeerrormessage-nanthrowrangeerrormessage-nanthrowerrorlocalvalue-nanthrowerrorlocalvalue-int)